### ISSUE TITLE
[FEATURE] Migrate to new unique databricks hostnames

### ISSUE THREAD

**Issue by sdebruyn** [databrickslabs/terraform-provider-databricks#34]
**Is your feature request related to a problem? Please describe.**
Currently not a problem, but it's advised to start using the new unique URLs for each Databricks workspace as documented in https://docs.microsoft.com/en-us/azure/databricks/release-notes/product/2020/april#unique-urls-for-each-azure-databricks-workspace

**Describe the solution you'd like**
Replace the current code that uses https://<region>.azuredatabricks.net/<workspaceid>

**Additional context**
The current hostnames have not been deprecated (yet) so we have still time.

**Comment by sdebruyn** [comment]
Tracked in the azurerm provider in https://github.com/terraform-providers/terraform-provider-azurerm/issues/6732

**Comment by nfx** [comment]
@sdebruyn we'll look into this issue and see what we can do. this is not a complicated one.

**Comment by lawrencegripper** [comment]
Created a PR here to add into AzureRM: https://github.com/terraform-providers/terraform-provider-azurerm/pull/6973

**Comment by stikkireddy** [comment]
@lawrencegripper also noticed that in the existing code I actually can get the url with the GET request from the management API so, we may not need the workspace url fetched from the Azure RM client as the management api provides a url with the new format. 

The following get request provides it:

```
https://management.azure.com/subscriptions/" + a.TokenPayload.SubscriptionID + "/resourceGroups/" + a.TokenPayload.ResourceGroup + "/providers/Microsoft.Databricks/workspaces/" + a.TokenPayload.WorkspaceName + "" +
		"?api-version=2018-04-01
```

**Comment by lawrencegripper** [comment]
Ah nice work around, in good news the change is shipping now from azurerm so can be used shortly if that saves some effort https://github.com/terraform-providers/terraform-provider-azurerm/issues/6732

**Comment by stuartleeks** [comment]
Just shipped 😀

https://github.com/terraform-providers/terraform-provider-azurerm/releases/tag/v2.11.0

**Comment by EliiseS** [comment]
I'll be picking up this one in an attempt that it'll fix #92
